app/experimenter/experiments/serializers/recipe.py

```diff
@@ -67,6 +67,21 @@ def get_channels(self, obj):
         return [obj.firefox_channel.lower()]
 
 
+class FilterObjectPlatformSerializer(serializers.ModelSerializer):
+    type = serializers.SerializerMethodField()
+    platforms = serializers.SerializerMethodField()
```

Comment:
I think you can omit this field definition and the method altogether?  If you're just returning the  field verbatim won't it just serialize it automatically?

Reply:
yes, woops.